dll/win32/shell32/folders/CDesktopFolder.cpp

```diff
@@ -448,12 +447,50 @@ HRESULT WINAPI CDesktopFolder::CompareIDs(LPARAM lParam, PCUIDLIST_RELATIVE pidl
 {
     bool bIsDesktopFolder1, bIsDesktopFolder2;
 
+    if (LOWORD(lParam) >= DESKTOPSHELLVIEWCOLUMNS)
```

Comment:
Why `LOWORD()`?

Reply:
I followed the code shown in CFSFolder.cpp here.
This was mostly because I thought that the programmer was more knowledgeable than me: 
https://github.com/reactos/reactos/blob/dd54cd32cf02254e9fc9ef78eb544bd580cd8787/dll/win32/shell32/folders/CFSFolder.cpp#L945

Reply:
Well, then `if (lParam ==` too should use `LOWORD()`...

Reply:
lParam should be changed everywhere it was used to LOWORD(lParam). Thanks.

Reply:
This code has been restored to its original state.